docs/content/development/extensions-core/variance.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: doc_page
+---
+
+# Variance aggregator
+
+Used to get variance (or stddev via post aggregator) of numeric column. The algorithm used here is the same with variance udf of apache hive. See javadoc of io.druid.query.aggregation.variance.VarianceHolder for detail.
+
+Example:
```

Comment:
these are just examples. We also need docs describing what the aggregators are and what each field means.